.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
```

Comment:
tho' I guess this value does appear in [the babel-eslint readme](https://github.com/babel/babel-eslint#setup). hmmm.